File: asynchronous/src/main/java/com/dao/asynchronous/Conclude.java
package com.dao.asynchronous;

import androidx.annotation.NonNull;
import androidx.fragment.app.FragmentManager;

/**
 * Created in 12/11/18 14:14.
 *
 * @author Diogo Oliveira.
 */
public interface Conclude
{
    void cancel();

    boolean isAttach();

    void finish(@NonNull FragmentManager manager);
}

File: asynchronous/src/main/java/com/dao/asynchronous/Conclude.kt
package com.dao.asynchronous

/**
 * Created in 31/01/19 19:59.
 *
 * @author Diogo Oliveira.
 */
interface Conclude : AsyncFinished
{
    fun isAttach(): Boolean

    fun cancel(interruptIfRunning: Boolean = true): Boolean
}